src/custom/constants/chains/chainsMod.ts

```diff
@@ -11,92 +14,172 @@ export enum SupportedChainId {
+    rpcUrls: ['https://rinkeby.arbitrum.io/rpc'],
   }, */
   [SupportedChainId.MAINNET]: {
     docs: 'https://docs.uniswap.org/',
     explorer: 'https://gnosis-protocol.io/mainnet',
     infoLink: '',
-    // label: 'Mainnet',
     label: 'Ethereum',
-    logoUrl: EthereumLogo,
+    logoUrl: EthereumLogo, // mod
+    nativeCurrency: { name: 'ETH', symbol: 'ETH', decimals: 18 },
   },
   [SupportedChainId.RINKEBY]: {
     docs: 'https://docs.uniswap.org/',
     explorer: 'https://gnosis-protocol.io/rinkeby',
     infoLink: '',
     label: 'Rinkeby',
-    // logoUrl: EthereumLogo,
+    nativeCurrency: { name: 'Rinkeby ETH', symbol: 'rinkETH', decimals: 18 },
     logoUrl: RinkebyLogo, // mod
   },
-  [SupportedChainId.XDAI]: {
-    docs: 'https://docs.uniswap.org/',
-    explorer: 'https://gnosis-protocol.io/xdai',
-    infoLink: '',
-    label: 'xDai',
-    // logoUrl: EthereumLogo,
-    logoUrl: xDaiLogo, // mod
-  },
   /* [SupportedChainId.ROPSTEN]: {
     docs: 'https://docs.uniswap.org/',
     explorer: 'https://ropsten.etherscan.io/',
     infoLink: 'https://info.uniswap.org/#/',
     label: 'Ropsten',
+    nativeCurrency: { name: 'Ropsten ETH', symbol: 'ropETH', decimals: 18 },
   },
   [SupportedChainId.KOVAN]: {
     docs: 'https://docs.uniswap.org/',
     explorer: 'https://kovan.etherscan.io/',
     infoLink: 'https://info.uniswap.org/#/',
     label: 'Kovan',
+    nativeCurrency: { name: 'Kovan ETH', symbol: 'kovETH', decimals: 18 },
   },
   [SupportedChainId.GOERLI]: {
     docs: 'https://docs.uniswap.org/',
     explorer: 'https://goerli.etherscan.io/',
     infoLink: 'https://info.uniswap.org/#/',
     label: 'Görli',
+    nativeCurrency: { name: 'Görli ETH', symbol: 'görETH', decimals: 18 },
   },
   [SupportedChainId.OPTIMISM]: {
+    blockWaitMsBeforeWarning: ms`10m`,
     bridge: 'https://gateway.optimism.io/',
     docs: 'https://optimism.io/',
     explorer: 'https://optimistic.etherscan.io/',
-    infoLink: 'https://info.uniswap.org/#/optimism/',
-    label: 'Optimism',
+    infoLink: 'https://info.uniswap.org/#/optimism',
+    label: 'OΞ',
     logoUrl: optimismLogoUrl,
+    nativeCurrency: { name: 'Optimistic ETH', symbol: 'ETH', decimals: 18 },
+    rpcUrls: ['https://mainnet.optimism.io'],
+    statusPage: 'https://optimism.io/status',
   },
   [SupportedChainId.OPTIMISTIC_KOVAN]: {
+    blockWaitMsBeforeWarning: ms`10m`,
     bridge: 'https://gateway.optimism.io/',
     docs: 'https://optimism.io/',
     explorer: 'https://optimistic.etherscan.io/',
     infoLink: 'https://info.uniswap.org/#/optimism',
     label: 'Optimistic Kovan',
+    rpcUrls: ['https://kovan.optimism.io'],
     logoUrl: optimismLogoUrl,
+    nativeCurrency: { name: 'Optimistic kovETH', symbol: 'kovOpETH', decimals: 18 },
+    statusPage: 'https://optimism.io/status',
   }, */
+  [SupportedChainId.XDAI]: {
+    docs: 'https://docs.uniswap.org/',
+    explorer: 'https://gnosis-protocol.io/xdai',
```

Comment:
I know this was there already, but the explorer should be BlockScout rather than our explorer